python/cudf/cudf/core/_base_index.py

```diff
@@ -344,7 +344,7 @@ def names(self):
         """
         Returns a tuple containing the name of the Index.
```

Comment:
Make this match the pandas docstring, or make the type correct in the summary.
```suggestion
        Returns a FrozenList containing the name of the Index.
```